src/assets/404-tree.svg

```diff
@@ -0,0 +1,385 @@
+							/>
+					</g>
+					<g>
+						<path class="st2" d="M744.6,986.5v92.1c0,2.2,1.8,4,4,4s4-1.8,4-4v-92.1c0-2.2-1.8-4-4-4C746.3,982.5,744.6,984.3,744.6,986.5
+							z"/>
+					</g>
+					<g>
+						<path class="st2" d="M721.6,959.9v62.1c0,2.2,1.8,4,4,4s4-1.8,4-4v-62.1c0-2.2-1.8-4-4-4C723.4,955.9,721.6,957.7,721.6,959.9
+							z"/>
+					</g>
+					<g>
+						<path class="st2" d="M715.9,1244.4v52.1c0,2.2,1.8,4,4,4s4-1.8,4-4v-52.1c0-2.2-1.8-4-4-4S715.9,1242.2,715.9,1244.4z"/>
+					</g>
+					<g>
+						<path class="st2" d="M721.6,690.8v52.1c0,2.2,1.8,4,4,4s4-1.8,4-4v-52.1c0-2.2-1.8-4-4-4C723.4,686.8,721.6,688.6,721.6,690.8
+							z"/>
+					</g>
+					<g>
+						<path class="st2" d="M744.6,1229.4v22.1c0,2.2,1.8,4,4,4s4-1.8,4-4v-22.1c0-2.2-1.8-4-4-4
+							C746.3,1225.4,744.6,1227.2,744.6,1229.4z"/>
+					</g>
+					<g>
+						<path class="st2" d="M715.9,1099.5v22.1c0,2.2,1.8,4,4,4s4-1.8,4-4v-22.1c0-2.2-1.8-4-4-4S715.9,1097.2,715.9,1099.5z"/>
+					</g>
+					<g>
+						<path class="st2" d="M747.7,899.6v15.1c0,2.2,1.8,4,4,4s4-1.8,4-4v-15.1c0-2.2-1.8-4-4-4S747.7,897.4,747.7,899.6z"/>
+					</g>
+					<g>
+						<path class="st2" d="M747.7,743.7v99.1c0,2.2,1.8,4,4,4s4-1.8,4-4v-99.1c0-2.2-1.8-4-4-4S747.7,741.5,747.7,743.7z"/>
+					</g>
+				</g>
+			</g>
+			<g>
+				<polyline class="st0" points="616.1,421.9 465.2,378.6 465.2,424.1 616.1,467.4 				"/>
+				<path class="st2" d="M465.2,378.6v14l137.4,39.5c6.7,1.9,13.5-3.1,13.5-10.1L465.2,378.6z"/>
+				<polyline class="st1" points="616.1,421.9 465.2,378.6 465.2,424.1 616.1,467.4 				"/>
+				<g>
+					<path class="st2" d="M565.2,425.4c-0.4,0-0.7-0.1-1.1-0.2l-26.2-7.5c-2.1-0.6-3.4-2.8-2.7-4.9c0.6-2.1,2.8-3.3,4.9-2.7
+						l26.2,7.5c2.1,0.6,3.4,2.8,2.7,4.9C568.6,424.3,567,425.4,565.2,425.4z"/>
+				</g>
+				<g>
+					<path class="st2" d="M547.3,438.7c-0.4,0-0.7-0.1-1.1-0.2l-46.3-13.3c-2.1-0.6-3.4-2.8-2.7-4.9c0.6-2.1,2.8-3.3,4.9-2.7
+						l46.3,13.3c2.1,0.6,3.4,2.8,2.7,4.9C550.7,437.6,549.1,438.7,547.3,438.7z"/>
+				</g>
+			</g>
+			<g>
+				<g>
+					<polyline class="st0" points="872.5,836.9 985.2,804.6 985.2,836.9 872.5,869.3 					"/>
+					<path class="st2" d="M872.5,836.9c0,5,4.8,8.6,9.6,7.2l103.1-29.6v-10L872.5,836.9z"/>
+					<g>
+						<path class="st2" d="M956.9,835.1c-1.7,0-3.3-1.1-3.8-2.9c-0.6-2.1,0.6-4.3,2.7-4.9l16.9-4.8c2.1-0.6,4.3,0.6,4.9,2.7
+							c0.6,2.1-0.6,4.3-2.7,4.9l-16.9,4.8C957.7,835.1,957.3,835.1,956.9,835.1z"/>
+					</g>
+					<g>
+						<path class="st2" d="M919.2,847.3c-1.7,0-3.3-1.1-3.8-2.9c-0.6-2.1,0.6-4.3,2.7-4.9l15.9-4.6c2.1-0.6,4.3,0.6,4.9,2.7
+							c0.6,2.1-0.6,4.3-2.7,4.9l-15.9,4.6C919.9,847.2,919.5,847.3,919.2,847.3z"/>
+					</g>
+					<polyline class="st1" points="872.5,836.9 985.2,804.6 985.2,836.9 872.5,869.3 					"/>
+				</g>
+				<g>
+					<text transform="matrix(1.0085 0 0 1 969.4008 838.2935)" class="st3 st4 st5"></text>
```

Comment:
Not all systems can display the  character

![image](https://user-images.githubusercontent.com/12701942/42233260-589a0f80-7ef1-11e8-9100-5b966c2029d7.png)